docs/plugins/delete-by-query.asciidoc

```diff
@@ -1,270 +0,0 @@
-[[plugins-delete-by-query]]
-=== Delete By Query Plugin
```

Comment:
I'd add a section with the `[[plugins-delete-by-query]]` ID to the redirects page: https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/plugins/redirects.asciidoc

(but first build --all the docs to ensure that you've caught any places linking to the DBQ plugin)